src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -62,17 +81,20 @@ export class EntityCountDashboardComponent
    *
    * Default is "center".
    */
-  @Input() groupBy = "center";
+  @Input() groupBy: string[] = ["center", "gender"];
```

Comment:
We'll need a (very simple) config migration that changes groupBy if it's a string and wraps it as a single item in an array (for existing configs in our client's databases).

@sadaf895 , can you try to write a unit test and then a migration function for this in our ConfigService?